justfile

```diff
@@ -81,6 +81,7 @@ clean:
 lint:
     #!/usr/bin/env sh
     set -e
+    ./bin/lint_ignore
```

Comment:
This should be inlined so its easier to copy justfile between machines/branches.